### PR TITLE
Added optional HTTPS configurations

### PR DESCRIPTION
**Pull request by BClark09**
Based on feedback I received and previously made questions around the community forums. Plus corrected spelling mistakes.

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
First time using the "Request feedback" feature. Well wow this looks scary :smiley:

**Comment by BClark09** [comment]
:D All done, thanks again! I hope that's better, I didn't even realised there was syntax highlighting built in!

File: configuration/nginx.md [hunk 1]
```diff
@@ -267,3 +275,38 @@ server {
...
+	ssl_protocols                   TLSv1 TLSv1.1 TLSv1.2;
+	ssl_prefer_server_ciphers       on;
+	ssl_dhparam                     /etc/nginx/ssl/dhparam.pem;
+	ssl_ciphers                     ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA:HIGH:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!CBC:!EDH:!kEDH:!PSK:!SRP:!kECDH;
+	ssl_session_timeout             1d;
+	ssl_session_cache               shared:SSL:10m;
+	keepalive_timeout               70;
+```
+
+Feel free to test the new configuration again on [SSL Labs](https://www.ssllabs.com/ssltest/). If you're achieving A or A+ here, then your client-openHAB communication is very secure.
```
Comment (ThomDietrich) [inline configuration/nginx.md]: "very secure" :smile_cat:

**Comment by ThomDietrich** [comment]
You have a very nice writing style. If you'd like, it would be great if you could continue writing for openHAB ;)

**Comment by BClark09** [comment]
Thanks @ThomDietrich! I'll be sure to help where I can, but can't promise anything. I'm in a crucial stage of a PhD :fearful:  so really have the weekends and the odd lunch break to look at other things.

**Pull request merged**